pkg/services/sqlstore/migrations/dashboard_public_config_mig.go

```diff
@@ -8,19 +8,26 @@ func addPublicDashboardMigration(mg *Migrator) {
 	var dashboardPublicCfgV1 = Table{
 		Name: "dashboard_public_config",
 		Columns: []*Column{
-			{Name: "uid", Type: DB_BigInt, IsPrimaryKey: true},
+			{Name: "uid", Type: DB_NVarchar, Length: 40, IsPrimaryKey: true},
 			{Name: "dashboard_uid", Type: DB_NVarchar, Length: 40, Nullable: false},
 			{Name: "org_id", Type: DB_BigInt, Nullable: false},
+			{Name: "time_variables", Type: DB_Text, Nullable: false},
```

Comment:
Can we make this into settings json field so we can use it for more than just time variables?

Reply:
We don't have a schema to validate the dashboard blob or the settings blob against so want to rely on the database schema to help us handle validations. 

Doing it this way allows us to handle combining data with explicit validations on demand instead of needing to parse and traverse a large json blob containing everything.

We can change this field to be Json, but I want to keep them separate until we need to combine them.

Reply:
@jalevin not sure I understand, so your saying time_variables is not a json blob? is it a single value? a comma separated list?

Reply:
I misread this as requesting a change for the database column to be jsonb. It will be json stored in this text field.